components/console-backend-service/internal/domain/k8s/limitrange_resolver_test.go

```diff
@@ -14,7 +14,7 @@ func TestLimitRangeResolver_LimitRangeQuery(t *testing.T) {
 	// GIVEN
 	informer := fixLimitRangeInformer(fixLimitRange())
 
-	svc := newLimitRangeService(informer)
+	svc := newLimitRangeService(informer, nil)
 	testingUtils.WaitForInformerStartAtMost(t, time.Second, informer)
 
 	resolver := newLimitRangeResolver(svc)
```

Comment:
You're missing a test for creating LimitRange

Reply:
added